src/your-first-flutter-app-to-appstore/index.md

```diff
@@ -51,14 +51,13 @@ In this tutorial, you will be going through all the steps necessary to build, si
 
```

Comment:
Towards the beginning of the document, I'd make a couple more edits.
"we will use Codemagic's Workflow Editor, though you can switch to using YAML for your projects workflow configuration afterwards if you wish" -> "you project's workflow"

"How to setup an account on Apple App Store connect" -> "How to set up..."